main.py

```diff
@@ -0,0 +1,18 @@
+import Learner_Functions as fl

```

Comment:
Would it make sense to remove this file and put the code in a `if __name__ == "__main__":` block in `Learner_Functions.py` instead?